app/deliver_grant_funding/templates/deliver_grant_funding/grant_setup/description.html

```diff
@@ -0,0 +1,61 @@
+{% extends "common/base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
+
+{% block pageTitle %}
+  {{ form.description.label.text }}
```

Comment:
One that we can address separately but the titles should keep the service name in. We're probably not consistent about this yet so I'll raise a separate ticket for it.